tests/attr/test_input_x_gradient.py

```diff
@@ -96,14 +101,16 @@ def _input_x_gradient_classification_assert(self, nt_type="vanilla"):
             output = model(input)[:, target]
             output.backward()
             expercted = input.grad * input
-            self.assertEqual(
-                expercted.detach().numpy().tolist(),
-                attributions.detach().numpy().tolist(),
-            )
+            if isinstance(attributions, Tensor):
```

Comment:
added if check here and below, otherwise ./scripts/run_mypy.sh gives errors on 
```

tests/attr/test_input_x_gradient.py:87: error: Item "Tuple[Tensor, ...]" of "Union[Tensor, Tuple[Tensor, ...]]" has no attribute "shape"
tests/attr/test_input_x_gradient.py:107: error: Item "Tuple[Tensor, ...]" of "Union[Tensor, Tuple[Tensor, ...]]" has no attribute "detach"
```

due to `inputs: Union[Tensor, Tuple[Tensor, ...]]`

Reply:
The change above to TensorOrTupleOfTensors should avoid this issue, and you can remove this and the extra case below.